diff -uNr hhvm-ext-geoip-centos/CMake/GeoIP.cmake hhvm-ext-geoip/CMake/GeoIP.cmake
--- hhvm-ext-geoip-centos/CMake/GeoIP.cmake     2014-07-24 14:11:27.731127404 -0400
+++ hhvm-ext-geoip/CMake/GeoIP.cmake    2014-07-24 14:11:42.166127677 -0400
@@ -22,7 +22,7 @@

 # Find library that contains version number in the file name.
 execute_process(
-    COMMAND find ${GEOIP_LIB_DIRECTORY} -name "libGeoIP.*.*.*.*"
+    COMMAND find /usr/${GEOIP_LIB_DIRECTORY} -name "libGeoIP.*.*.*.*"
     COMMAND cut -d . -f 2-5
     COMMAND sort
     COMMAND tail -1

