diff --git a/CMake/FindLibSQLite.cmake b/CMake/FindLibSQLite.cmake
index 2567fe9..bdd86af 100644
--- a/CMake/FindLibSQLite.cmake
+++ b/CMake/FindLibSQLite.cmake
@@ -8,12 +8,7 @@
 find_package(PkgConfig)
 pkg_check_modules(PC_SQLITE3 QUIET sqlite3)
 
-if (PC_SQLITE3_FOUND AND NOT (PC_SQLITE3_VERSION VERSION_LESS "3.8.0"))
-  set(LIBSQLITE3_FOUND 0)
-  set(LIBSQLITE3_INCLUDE_DIR "LIBSQLITE3_INCLUDE_DIR-NOTFOUND")
-  set(LIBSQLITE3_LIBRARY "LIBSQLITE3_LIBRARY-NOTFOUND")
-  message(STATUS "Found non-working version of libsqlite3")
-else()
+if (PC_SQLITE3_FOUND)
   find_path(LIBSQLITE3_INCLUDE_DIR
             NAMES sqlite3.h
             HINTS ${PC_SQLITE3_INCLUDE_DIRS})
@@ -25,10 +20,7 @@ else()
                                     LIBSQLITE3_LIBRARY
                                     LIBSQLITE3_INCLUDE_DIR)
 
-endif (PC_SQLITE3_FOUND AND NOT (PC_SQLITE3_VERSION VERSION_LESS "3.8.0"))
-
-if (NOT LIBSQLITE3_FOUND)
-  message(STATUS "Using third-party bundled libsqlite3")
-else()
   message(STATUS "Found libsqlite3: ${LIBSQLITE3_LIBRARY}")
-endif (NOT LIBSQLITE3_FOUND)
+else()
+  message(STATUS "Using third-party bundled libsqlite3")
+endif (PC_SQLITE3_FOUND)
