diff --git a/hphp/test/quick/array_plus_eq_dtor.php b/hphp/test/quick/array_plus_eq_dtor.php
deleted file mode 100644
index 0e03866..0000000
--- a/hphp/test/quick/array_plus_eq_dtor.php
+++ /dev/null
@@ -1,11 +0,0 @@
-<?php
-
-class dtor { public function __destruct() { echo "dtor\n"; } }
-
-function main() {
-  $a = array(new dtor);
-  var_dump($a+array(4,5,6));
-}
-
-main();
-echo "done\n";
diff --git a/hphp/test/quick/array_plus_eq_dtor.php.expect b/hphp/test/quick/array_plus_eq_dtor.php.expect
deleted file mode 100644
index fa90ff9..0000000
--- a/hphp/test/quick/array_plus_eq_dtor.php.expect
+++ /dev/null
@@ -1,11 +0,0 @@
-array(3) {
-  [0]=>
-  object(dtor)#1 (0) {
-  }
-  [1]=>
-  int(5)
-  [2]=>
-  int(6)
-}
-dtor
-done
